test/fixtures/taxable_taxonomies.yml

```diff
@@ -133,3 +133,13 @@ twentysix:
   taxonomy: organization2
   taxable: Location 2
   taxable_type: Taxonomy
+
+restricted_user_organization1:
+  taxonomy: organization1
+  taxable: restricted
+  taxable_type: User
+
+restricted_user_location1:
+  taxonomy: location1
+  taxable: restricted
+  taxable_type: User
```

Comment:
I dug into one of the failures and adding these is actually the cause of the katello error.  In a lot of our tests, we are using the restricted user and creating filters without any organization or location.  This worked previously due to this line:

https://github.com/theforeman/foreman/blob/develop/app/models/concerns/taxonomix.rb#L186

but now that the user is restricted to a particular org/loc, each filter must have at least one of the user's orgs and locations.  In our tests we're using Role#add_add_permissions! to setup the role to test, and that doesn't seem to be able to handle orgs/locs at all.